magpie/utils.py

```diff
@@ -5,6 +5,7 @@
 except ImportError:
```

Comment:
Ah yeah, I forgot to describe that. So in `magpie/utils.py` there are a couple of functions that are not directly called anywhere, but might be useful for people using the software for analysis of their corpus.

One is for computing the frequency of classes in a corpus other gives you top N classes in the corpus and things like that. I decided not to remove them, because they might come in handy for somebody, so I just updated them and their docs.